documentation/docs/scenarios/changeManagement.md

```diff
@@ -0,0 +1,343 @@
+                           ]
+                         ]
+```
+
+## (Optional) Create a Transport Request (`transportRequestCreate`)
+
+Create a transport request for a change document in SAP Solution Manager.
+
+### Context
+
+Depending on your workflow, this step is optional. If you already have a transport request, define it in the commit message. For example:
+
+```
+Lorem ipsum dolor sit amet, cum sucipat
+
+    sed diam nonumy eirmod tempor invidunt ut labore
+    et dolore magna aliquyam erat, sed diam voluptua.
+    At vero eos et accusam et justo duo
+    dolores et ea rebum nisi bene.
+
+    TransportRequest: ZZZDK900026
+```
+Per default, it is expected that one of the commits between origin/master and the HEAD branch contains this transport request ID.
+
+### Mandatory parameters
+
+| Parameter | Description |
+| --- | --- |
+| `script` | The common script envoronment of the running Jenkinsfile. The reference to the script that calls the pipeline step is privided by the `this` parameter, as in `script: this`. This allows the function to access the `commonPipelineEnvironment` to retrieve configuration parameters. |
+| `changeManagement/credentialsId` |  The ID of the credentials that are required to connect to SAP Solution Manager. |
+| `changeManagement/endpoint` | The address of SAP Solution Manager. |
+
+For an overview of the optional parameters, see **Parameters** in [transportRequestCreate](https://sap.github.io/jenkins-library/steps/transportRequestCreate/).
+
+### Step configuration
+
+The step is configured by using a customer configuration file that is provided as a resource in a custom shared library.
+```
+@Library('piper-library-os@master') _
+
+// the shared lib containing the additional configuration
+// must be configured in Jenkins
+@Library(foo@master') __
+
+// in the shared lib denoted by 'foo', the additional configuration file
+// must be located under 'resources' ('resoures/myConfig.yml')
+prepareDefaultValues script: this, customDefaults: 'myConfig.yml'
+```
+Example content of `resources/myConfig.yaml` in branch `master` of the repository denoted by `foo`:
+```
+general:
+  changeManagement:
+    changeDocumentLabel: 'ChangeDocument\s?:'
+    cmClientOpts: '-Djavax.net.ssl.trustStore=<path to truststore>'
+    credentialsId: 'CM'
+    type: 'SOLMAN'
+    endpoint: 'https://example.org/cm'
+    git:
+      from: 'HEAD~1'
+      to: 'HEAD'
+      format: '%b'
+```
+The properties configured in section `general/changeManagement` are shared between all steps related to change management.
+You can also configure the properties on a per-step basis, for example:
+```
+[...]
+ steps:
+   transportRequestCreate:
+     changeManagement:
+       type: 'SOLMAN'
+       endpoint: 'https://example.org/cm'
+       [...]
+```
+The parameters can also be provided when the step is invoked. See [Examples](#Examples).
+
+### Result
+
+As a result, you get the ID of the newly created transport request.
```

Comment:
In the meantime we do not support return values from the steps. The transport request Id is made available via the `commonPipelineEnvironment`. Parameter: `transportRequestId`. Maybe something like: "The ID of the newly created transport request is available via parameters 'transportRequestId' in the 'commonPipelineEnvironment'."